include/fcl/BV/RSS.h

```diff
@@ -111,7 +111,7 @@ class RSS
   /// @brief Volume of the RSS
   inline FCL_REAL volume() const
   {
-    return (l[0] * l[1] * 2 * r + 4 * boost::math::constants::pi<FCL_REAL>() * r * r * r);
+    return (l[0] * l[1] * 2 * r + 4 * M_PI * r * r * r);
```

Comment:
> M_PI works with visual studio if you define _USE_MATH_DEFINES

Thanks -- I know about that but it's a non-standard hack so I think Mark's `fcl::constants::pi` idea is better.